tests/jobs.py

```diff
@@ -901,80 +901,6 @@ def do_schedule(function, function2):
         do_schedule()
         self.assertEquals(2, len(executor.queued_tasks))
 
-    def test_retry_still_in_executor(self):
```

Comment:
This has been removed because it was relying on a failed dagrun still getting it's TIs executed. Only running dagruns should have their TIs executed.